playbook/update_repo.yaml

```diff
@@ -112,26 +112,30 @@
       failed_when: git_change.rc == 0 # fail when there are no changes
 
 
-# Check if anything was changed, if yes commit changes and create a pull request, otherwise skip rest of this play
+# Commit changes
```

Comment:
I was proud of my comment actually stating that this influences further execution of the playbook ':)